scopes/webpack/webpack/webpack.dev-server.ts

```diff
@@ -1,23 +1,26 @@
 import { DevServer } from '@teambit/bundler';
 import { Server } from 'http';
-import webpack, { Configuration } from 'webpack';
-import WsDevServer, { Configuration as DevServerConfiguration } from 'webpack-dev-server';
+import { Configuration } from 'webpack';
+import { Configuration as DevServerConfiguration } from 'webpack-dev-server';
 
 export interface WebpackConfigWithDevServer extends Configuration {
   devServer: DevServerConfiguration;
 }
 export class WebpackDevServer implements DevServer {
-  constructor(private config: WebpackConfigWithDevServer) {}
+  constructor(private config: WebpackConfigWithDevServer, private webpack, private WsDevServer) {}
 
-  private getCompiler() {
-    return webpack(this.config);
+  private getCompiler(): any {
+    return this.webpack(this.config);
```

Comment:
don't we want to have a fallback to the webpack as it was before in case it wasn't provided?